From bce62e05fef9bf885fe5904b6830319063b05349 Mon Sep 17 00:00:00 2001
From: "Guoniu.Zhou" <guoniu.zhou@nxp.com>
Date: Fri, 14 Oct 2016 11:20:03 +0800
Subject: [PATCH 1214/5242] MLK-13341 IPU: mxc_vout: fix the potential
 uninitalized variable usage

commit  c115c88362244d320c64e1ac303bcbee6b685f5a from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix coverity CID 17624 uninitialized scalar variable

The 'fb_fmt' variable may be used before uninitialized
So initialize it at the begining.

Signed-off-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/media/platform/mxc/output/mxc_vout.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/mxc/output/mxc_vout.c b/drivers/media/platform/mxc/output/mxc_vout.c
index 293b076..b557efd 100644
--- a/drivers/media/platform/mxc/output/mxc_vout.c
+++ b/drivers/media/platform/mxc/output/mxc_vout.c
@@ -304,7 +304,7 @@ static ipu_channel_t get_ipu_channel(struct fb_info *fbi)
 static unsigned int get_ipu_fmt(struct fb_info *fbi)
 {
 	mm_segment_t old_fs;
-	unsigned int fb_fmt;
+	unsigned int fb_fmt = 0;
 
 	if (fbi->fbops->fb_ioctl) {
 		old_fs = get_fs();
-- 
1.7.9.5

